packages/neuron-wallet/src/block-sync-renderer/sync/queue.ts

```diff
@@ -172,7 +206,11 @@ export default class Queue {
             }
           }
           await TransactionPersistor.saveFetchTx(tx)
-          await WalletService.updateUsedAddresses(addresses)
+          const anyoneCanPayBlake160s = anyoneCanPayInfos.map(info => info.blake160)
+          await WalletService.updateUsedAddresses(addresses, anyoneCanPayBlake160s)
```

Comment:
I would be interested in knowing more about how the AddressDao works. Particularly how ACP related addresses are differentiated from the other addresses.

Maybe @classicalliu can clarify a bit on this one.